sw/device/lib/testing/clkmgr_testutils.c

```diff
@@ -0,0 +1,39 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/clkmgr_testutils.h"
+
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/rand_testutils.h"
+
+void clkmgr_testutils_check_trans_clock_status(
+    dif_clkmgr_t *clkmgr, dif_clkmgr_params_t params,
+    dif_clkmgr_hintable_clock_t clock, bool exp_clock_enabled) {
+  bool clock_enabled;
+  CHECK_DIF_OK(dif_clkmgr_hintable_clock_get_enabled(clkmgr, params, clock,
```

Comment:
minor nit: maybe rename `clock` parameter to `clk`? (GH seems to highlight `clock` as if it's a function, maybe because of libc clock() function? not that we implement that in our OT runtime lib, but would it would look a little cleaner).